src/StandardGovernor.sol

```diff
@@ -356,6 +356,8 @@ contract StandardGovernor is IStandardGovernor, BatchGovernor {
         uint8 support_,
         string memory reason_
     ) internal override {
+        if (weight_ == 0) return;
```

Comment:
redundant here, can be only in BatchGovernor

Reply:
It was mainly to avoid wasting more gas but I do agree it is redundant.